topydo/lib/DCommand.py

```diff
@@ -125,4 +123,8 @@ def _execute_multi_specific(self):
                 self.error(self.condition_failed_text())
 
         current_active = self._active_todos()
-        self._print_unlocked_todos(old_active, current_active)
+        self._delta = [todo for todo in current_active
+                       if todo not in old_active]
+
+    def post_archive(self):
```

Comment:
Minor: a method name with a verb is more readable: e.g. `execute_post_archive_actions`. Same comment for the new `Transaction` method below.